OpenSim/Common/Component.cpp

```diff
@@ -1053,6 +1067,17 @@ bool Component::constructOutputForStateVariable(const std::string& name)
     return constructOutput<double>(name, func, SimTK::Stage::Model);
 }
 
+// helper method to specify the order of subcomponents.
```

Comment:
`(append subcomponent to the end of Component's _orderedSubcomponents list if it isn't already in the list)`?